config_params/connection_params.py

```diff
@@ -0,0 +1,5 @@
+class ConnectionParams:
+    def __init__(self, system_write_conn_count, system_read_conn_count):
```

Comment:
nit: is "conn" a popular diminutive for connection? If not consider simply using "connection" or even omitting it?

Reply:
I think it is popular. But as it was unclear to you, will fix it.